benchmarks/operator_benchmark/pt/qrnn_test.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import operator_benchmark as op_bench
+import torch
+import torch.nn.quantized.dynamic as nnqd
+
+"""
+Microbenchmarks for RNNs.
+"""
+
+qrnn_configs = op_bench.config_list(
+    attrs=[
+        [1, 3, 1],
+        [5, 7, 4],
+    ],
+    # names: input_size, hiddent_size, num_layers
+    attr_names=["I", "H", "NL"],
+    cross_product_configs={
+        "B": (False, True),  # Bias
+        "D": (False, True),  # Bidirectional
+        "dtype": (torch.qint8, torch.qint32)  # dtype
+    },
+    tags=["short"]
+)
+
+class _RNNBenchmarkBase(op_bench.TorchBenchmarkBase):
+    def init(self, input_size, hidden_size, dtype):
+        print(input_size, hidden_size, dtype)
```

Comment:
This should be available in the printed result already?

Reply:
Still debugging -- I'll ping when ready for final review. Sorry about the confusion